### ISSUE TITLE
Parsing fails on valid JS file

### ISSUE THREAD

**Issue by bzz** [bblfsh/javascript-driver#56]
Discovered though https://github.com/bblfsh/javascript-driver/issues/54#issuecomment-462829594

Using latest bblfshd v2.11.6/client.v3/javasript driver v2.6.0 parsing of the valid [2.4 mb .js file](https://github.com/laravel/telescope/blob/6f0a10ec586cfa1a22218b6778bf9c1572b97912/public/app.js) fails.

## Steps to reproduce

```
wget https://raw.githubusercontent.com/laravel/telescope/6f0a10ec586cfa1a22218b6778bf9c1572b97912/public/app.js
node --check app.js // check JS syntax
go install github.com/bblfsh/client-go/cmd/bblfsh-cli/...
bblfsh-cli ./app.js
```

## Expected

Very long semantic UAST printed in YAML format. 

## Actual

```
couldn't parse public/app.js: transform failed: received 15 errors:
	check: key "extra": field was not used: parenthesized
	check: key "extra": field was not used: parenthesized
	check: key "extra": field was not used: parenthesized
	check: key "extra": field was not used: parenthesized
	check: key "extra": field was not used: parenthesized
	check: key "extra": field was not used: parenthesized
	check: key "extra": field was not used: parenthesized
	check: key "extra": field was not used: parenStart
	check: key "extra": field was not used: parenStart
	check: key "extra": field was not used: parenStart
	check: key "extra": field was not used: parenthesized
	check: key "extra": field was not used: parenthesized
	check: key "extra": field was not used: parenthesized
	check: key "extra": field was not used: parenthesized
	check: key "extra": field was not used: parenStart
```

As noted https://github.com/bblfsh/javascript-driver/issues/54#issuecomment-462853077 actual order of the failing Transformations can vary.

**Comment by juanjux** [comment]
Context: this is because SDK 2.14.0 added new checks to the semantic checker for silently dropped fields.

**Comment by bzz** [comment]
```
npm install @babel/parser
NODE_OPTIONS=--max_old_space_size=12000 ./node_modules/.bin/parser app.js  > app-js.ast
```

JavaScript: 2.5Mb
Native AST: 357Mb

😱

**Comment by creachadair** [comment]
> ```
> npm install @babel/parser
> NODE_OPTIONS=--max_old_space_size=12000 ./node_modules/.bin/parser app.js  > app-js.ast
> ```
> JavaScript: 2.5Mb
> Native AST: 357Mb
> 
> 😱

I'm not too surprised—though this is a more dramatic expansion than usual. Syntactic structure is expensive in general, and very much _not_ linear in file size. 🙃

**Comment by bzz** [comment]
> Syntactic structure is expensive in general, and very much _not_ linear in file size. 🙃
Indeed!

Just for the sake of experiment:
JavaScript: 2.5Mb
Native AST in JSON: 357Mb
bblfsh `.native` in YAML: 504Mb

**Comment by bzz** [comment]
This issue though is fixed in #57
